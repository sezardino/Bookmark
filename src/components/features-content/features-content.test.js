import React from 'react';
import {shallow} from 'enzyme';

import FeaturesContent from './features-content';

describe('Component FeaturesContent', () => {
	const mockData = [
		{id: 1, text: 'Text', img: './src/img.jpg', title: 'Title'},
		{id: 2, text: 'Text', img: './src/img.jpg', title: 'Title'},
		{id: 3, text: 'Text', img: './src/img.jpg', title: 'Title'},
	];
	const component = shallow(<FeaturesContent data={mockData} />);

	it('Component FeaturesContent rendered correctly', () => {
		expect(component.debug()).toMatchSnapshot();
	});
});
