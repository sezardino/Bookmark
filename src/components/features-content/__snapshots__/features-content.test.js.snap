// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component FeaturesContent Component FeaturesContent rendered correctly 1`] = `
"<Fragment>
  <ul className=\\"tabs features__tabs\\">
    <li tabIndex=\\"0\\" className=\\"tabs__item tabs__item--active\\" onClick={[Function: onClick]} />
    <li tabIndex=\\"0\\" className=\\"tabs__item \\" onClick={[Function: onClick]} />
    <li tabIndex=\\"0\\" className=\\"tabs__item \\" onClick={[Function: onClick]} />
  </ul>
  <ul className=\\"content features__content\\">
    <li className=\\"content__item content__item--active\\">
      <p className=\\"img-wrapper content__img pseudo pseudo--left\\">
        <img src=\\"./src/img.jpg\\" alt=\\"Title\\" />
      </p>
      <div className=\\"content__wrapper\\">
        <h2 className=\\"content__title title\\">
          Title
        </h2>
        <p className=\\"content__text text\\">
          Text
        </p>
        <Link to=\\"/extension\\" className=\\"button button--content button--blue content__button\\">
          More Info
        </Link>
      </div>
    </li>
    <li className=\\"content__item \\">
      <p className=\\"img-wrapper content__img pseudo pseudo--left\\">
        <img src=\\"./src/img.jpg\\" alt=\\"Title\\" />
      </p>
      <div className=\\"content__wrapper\\">
        <h2 className=\\"content__title title\\">
          Title
        </h2>
        <p className=\\"content__text text\\">
          Text
        </p>
        <Link to=\\"/extension\\" className=\\"button button--content button--blue content__button\\">
          More Info
        </Link>
      </div>
    </li>
    <li className=\\"content__item \\">
      <p className=\\"img-wrapper content__img pseudo pseudo--left\\">
        <img src=\\"./src/img.jpg\\" alt=\\"Title\\" />
      </p>
      <div className=\\"content__wrapper\\">
        <h2 className=\\"content__title title\\">
          Title
        </h2>
        <p className=\\"content__text text\\">
          Text
        </p>
        <Link to=\\"/extension\\" className=\\"button button--content button--blue content__button\\">
          More Info
        </Link>
      </div>
    </li>
  </ul>
</Fragment>"
`;
